test/execute.sh

```diff
@@ -2,7 +2,7 @@
 
 # Runs the executable and compares its output to the .expected file
 ./carp.sh $1 --log-memory -b && \
-  ./out/Untitled > test/output/$1.output.actual 2>&1
+  ~/.carp/out/Untitled > test/output/$1.output.actual 2>&1
```

Comment:
That's gonna break peoples/the CI run if they don't have their out folder setup like yours. Using `-x` like it use to be would solve that but I'm not sure how to not have the compiler output in the diff.

Reply:
Oh yeah, I didn’t want to commit that, sorry. I just need to do that to make the tests work on my machine atm.

Reply:
Maybe we should hardcode the out folder in the Carp tests for now.

Reply:
Right, but this script is also used for the examples. I’m just going to work on the CLI flag tonight I think.